src/hpx_init.cpp

```diff
@@ -781,7 +783,7 @@ namespace hpx
                 std::unique_ptr<hpx::runtime> rt(new runtime_type(cms.rtcfg_));
 
                 result = run_or_start(blocking, std::move(rt),
-                    cms, std::move(startup), std::move(shutdown));
+                    cms, std::move(params.startup), std::move(params.shutdown));
```

Comment:
Sorry for commenting only after this was merged... You can't move anything that has been marked `const`. I'm surprised clang-tidy didn't report this.